tests/end-to-end/external-solution-integration/pkg/testkit/clients.go

```diff
@@ -15,6 +15,7 @@ import (
 
 var (
 	apiRuleRes = schema.GroupVersionResource{Group: "gateway.kyma-project.io", Version: "v1alpha1", Resource: "apirules"}
+	function   = schema.GroupVersionResource{Group: "serverless.kyma-project.io", Version: "v1alpha1", Resource: "functions"}
```

Comment:
It's duplicated because it's in another test. External-solution represents a group of tests not a single test. And I followed the convention proposed above with `apiRuleRes`